test/cogctl_test.exs

```diff
@@ -561,14 +561,14 @@ defmodule CogctlTest do
     run("cogctl bundle install --enable --relay-groups=myrelays #{Path.join(@scratch_dir, "bundle4.yaml")}")
 
     assert run("cogctl bundle info bundle4") =~ ~r"""
-    Bundle ID:           .*
-    Version ID:          .*
-    Name:                bundle4
-    Installed Versions:  0.0.1
-    Status:              Enabled
-    Version:             0.0.1
-    Commands:            bar
-    Relay Groups:        myrelays
+    Bundle ID:     .*
+    Version ID:    .*
+    Name:          bundle4
+    Versions:      0.0.1
+    Status:        Enabled
+    Version:       0.0.1
```

Comment:
Now that we've changed `Installed Versions` to `Versions` I think it'd be more clear if we changed this to `Enabled Version`.